### ISSUE TITLE
Possibility for 'gg_viewfinder' function?

### ISSUE THREAD

**Issue by thebioengineer** [thebioengineer/camcorder#55]
It appears folks like to use gg_record for the side effect of viewing properly formatted displays. Might be beneficial of enabling that directly, no saving images directly - and another function to enable saving the final image later?

**Comment by gkaramanis** [comment]
Yeah, I would use it as well. I recall starting two quick projects where I would like to have the option to just view without saving.
